### PR TITLE
changed to referencing C APIs by empty brackets (Fortran)

### PR DESCRIPTION
**Pull request by brtnfld**
Also update HL /note to /attention.

### CODE REVIEW

**Review by byrnHDF — APPROVED** [review]

File: fortran/src/H5Pff.F90 [hunk 1]
```diff
@@ -948,7 +948,7 @@ END SUBROUTINE h5pget_alignment_f
 !! \param backing_store Boolean flag indicating whether to write the file contents to disk when the file is closed.
 !! \param hdferr        \fortran_error
 !!
-!! See C API: @ref herr_t H5Pget_fapl_core(hid_t fapl_id, size_t *increment, hbool_t *backing_store);
+!! See C API: @ref H5Pget_fapl_core()
```
Comment (lrknox) [inline fortran/src/H5Pff.F90]: should this be "See C API: @ref H5P**s**et_fapl_core()"? (SUBROUTINE is h5p**s**et_fapl_core_f, not **g**et).

Comment (brtnfld) [reply fortran/src/H5Pff.F90]: fixed.

File: fortran/src/H5Pff.F90 [hunk 2]
```diff
@@ -3412,7 +3412,7 @@ END SUBROUTINE h5pset_obj_track_times_f
 !! \param crt_intermed_group Specifies whether to create intermediate groups upon the creation of an object.
 !! \param hdferr             \fortran_error
 !!
-!! See C API: @ref herr_t H5Pget_create_intermediate_group(hid_t plist_id, unsigned *crt_intmd);
+!! See C API: @ref H5Pget_create_intermediate_group()
```
Comment (lrknox) [inline fortran/src/H5Pff.F90]: H5Pset?

**Review by lrknox — APPROVED** [review]

File: fortran/src/H5Pff.F90 (hunk 2 above)
Comment (brtnfld) [reply fortran/src/H5Pff.F90]: fixed.

**Pull request merged**